Cargo.toml

```diff
@@ -1,7 +1,23 @@
 [package]
 name = "ion-schema"
+authors = ["Amazon Ion Team <ion-team@amazon.com>"]
+description = "Implementation of Amazon Ion Schema"
+homepage = "https://github.com/amzn/ion-schema-rust"
+repository = "https://github.com/amzn/ion-schema-rust"
+license = "Apache-2.0"
+readme = "README.md"
+keywords = ["ion", "schema"]
+exclude = [
+    "**/.git/**",
+    "**/.github/**",
+    "**/.travis.yml",
+    "**/.appveyor.yml",
```

Comment:
These are unnecessary because we're not using any of these kinds of files in this repo.
```suggestion
```